src/Execute/executeQuickInput.ts

```diff
@@ -58,11 +58,11 @@ export async function runInferenceQuickInput(context: vscode.ExtensionContext) {
   async function selectInputModel(input: MultiStepInput, state: Partial<State>) {
     if (state.backend === undefined) {
       state.error = 'Backend to infer is not chosen. Please check once again.';
-      return;
+      return undefined;
```

Comment:
Aha, I got it.

This function returns sth on below code
```ts
    return (input: MultiStepInput) => selectInputSpec(input, state);
```

So returning `undefined` explicitly is important.